yellow-paper/docs/state/index.md

```diff
@@ -155,3 +186,9 @@ State *-- ContractTree : contract_tree
 import DocCardList from '@theme/DocCardList';
 
 <DocCardList />
+
+---
+
+:::warning **Discussion Point**: 
+"Indexed merkle tree" is not a very telling name, as our normal merkle trees are indexed too. I propose we call them "successor merkle trees" instead since each leaf refers to its successor. The low-nullifiers are also the predecessor of the nullifier you are inserting, so it seems nice that you prove that the nullifier you are inserting has a predecessor and that the predecessors successor would also be the successor of the nullifier you are inserting.
```

Comment:
'successor merkle tree' sounds like the successor of whatever merkle tree came before it 🤷

Reply:
"Sorted linked merkle tree"